sqlitecluster/SQLite.cpp

```diff
@@ -357,14 +357,16 @@ int SQLite::commit() {
     if (newJournalSize > _maxJournalSize) {
         // Delete the oldest entry
         uint64_t before = STimeNow();
-        string query = "DELETE FROM " + _journalName + " WHERE id < (SELECT MAX(id) FROM " + SQ(_journalName) + ") - " + SQ(_maxJournalSize);
+        string query = "DELETE FROM " + _journalName + " "
+                       "WHERE id < (SELECT MAX(id) " "FROM " + _journalName + ") - " + SQ(_maxJournalSize) + " "
+                       "LIMIT 10";
         SASSERT(!SQuery(_db, "Deleting oldest journal rows", query));
 
         // Figure out the new journal size.
         SQResult result;
-        SASSERT(!SQuery(_db, "getting commit min", "SELECT MIN(id) AS id FROM " + SQ(_journalName), result));
```

Comment:
Because it generates weird-looking queries that look like this:
```
SELECT a, b FROM 'table' WHERE condition;
```

Which is valid but strange when people are used to:
```
SELECT a, b FROM table WHERE condition;
```

And the journal name is not user-supplied and guaranteed to not need escaping.